source/config/configSpec.py

```diff
@@ -8,6 +8,31 @@
 from io import StringIO
 from configobj import ConfigObj
 
+DEFAULT_TEXT_PARAGRAPH_REGEX = (

```

Comment:
I think this should be defined elsewhere, perhaps in a sibling module named `configDefaults` or `configStrings` like `configFlags`

Reply:
I vote for `configDefaults` as this regex is not supposed to be used outside of the config spec.